stan/math/prim/mat/prob/bernoulli_logit_glm_lpmf.hpp

```diff
@@ -117,8 +117,7 @@ typename return_type<T_x, T_alpha, T_beta>::type bernoulli_logit_glm_lpmf(
 
   // Compute the necessary derivatives.
   operands_and_partials<T_x, T_alpha, T_beta> ops_partials(x, alpha, beta);
-  if (!is_constant_struct<T_beta>::value || !is_constant_struct<T_x>::value
-      || !is_constant_struct<T_alpha>::value) {
+  if (!is_constant_struct<T_beta, T_x, T_alpha>::value) {
```

Comment:
[note---no action]
All of the use here seems to be `!is_constant_struct`.  I can't think of anything shorter that makes sense---maybe `non_const`.